eng/Publishing.props

```diff
@@ -123,6 +128,33 @@
       PublishingVersion="3"
       IsStableBuild="$(IsStableBuild)" />
 
+    <PushToBuildStorage
```

Comment:
Just wondering, why do we now have both a `PushToAzureDevOpsArtifacts` and a `PushToBuildStorage` task? I thought the latter should replace the former?

Reply:
I should probably add a comment to clarify this.

Reply:
The build is already failing because of the missing `PushToBuildStorage` task.

Reply:
I need to update the condition to use the new task in VMR only.

Reply:
Fixed with https://github.com/dotnet/sdk/pull/39311/commits/e90fb536f5b2b071ece83a2cf4d4e63d203e28d9